src/app/components/card/card.component.html

```diff
@@ -0,0 +1,15 @@
+<mat-card class="c-card">
+    <img  class="c-card__image" src={{listing.images[0]}} alt="Card photo">
+    <mat-card-content class="c-card__content">
+      <h1 class="c-card__content--title">{{listing.title}}</h1>
+      <!-- <h4 class="c-card__content--description">{{listing.description}}</h4> -->
```

Comment:
Don't forget to remove unnecessary comments if you don't need some pieces of code